src/modules/AutoinstSoftware.rb

```diff
@@ -1020,19 +1017,19 @@ def ReadHelper
       Pkg.SourceStartManager(true)
       Pkg.PkgSolve(false)
 
-      all_patterns = Pkg.ResolvableProperties("", :pattern, "")
-      @all_xpatterns = Pkg.ResolvableDependencies("", :pattern, "")
+      all_patterns = Y2Packager::Resolvable.find(kind: :pattern)
```

Comment:
Um, do we actually need the `all_patterns` variable? To me it seems we could simply use `@all_patterns` in the `filter` call and remove it...